### ISSUE TITLE
[FR] Align file icons and colors

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/ios-app#1397]
**Problem**
Icons and Icon colors are quite different between web and mobile clients. This is particularly noticeable when I e.g. do presentations, see screenshot.

![screenshot_003187](https://github.com/user-attachments/assets/18053ee2-d336-4d81-9097-f3a905f7c331)

**Proposed solution**

Align icon style and colors between web and mobile clients.

![ios](https://github.com/user-attachments/assets/42fa3c8b-60e3-4c77-9c0e-8167b84b3d32)

@felix-schwarz @hosy cc @jesmrec Whats you opinion - do you consider the proposal an improvement?

**Comment by jesmrec** [comment]
For me, that's a good improvement in terms of product. Just need the new stuff to include, i guess

**Comment by hosy** [comment]
Yes, I absolutely agree that we should use the same icons across the platform. Where can I find the resources? @tbsbdr

**Comment by tbsbdr** [comment]
Ok, cool 👍 Please see https://github.com/owncloud/android/issues/4483#issuecomment-2389267111 for icon assets. @kulmann and team can provide more details if needed.
